nav2_smac_planner/src/analytic_expansion.cpp

```diff
@@ -292,6 +277,86 @@ typename AnalyticExpansion<NodeT>::AnalyticExpansionNodes AnalyticExpansion<Node
   return possible_nodes;
 }
 
+template<typename NodeT>
+void AnalyticExpansion<NodeT>::refineAnalyticPath(
+  const NodePtr & goal_node,
+  const NodeGetter & getter,
+  NodePtr & node,
+  AnalyticExpansionNodes & analytic_nodes,
+  float & best_score
```

Comment:
To that end, `best_score` should be a return type (it doesn't need to have input argument at all)